### PR TITLE
Ability to change the log level at runtime using jconsole

### PR DESCRIPTION
**Pull request by REASY**
- Change the log level at runtime and monitor main JVM metrics via [jconsole](https://docs.oracle.com/javase/7/docs/technotes/guides/management/jconsole.html)

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1949)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -565,10 +565,15 @@ def jfrWithMem = ["-XX:+UnlockCommercialFeatures", "-XX:+UnlockDiagnosticVMOptio
                   "-XX:StartFlightRecording=delay=2s,duration=60m,name=mem_ex,filename=recording.jfr,settings=profile_heap_exception",
                   "-XX:+FlightRecorder", "-XX:FlightRecorderOptions=disk=true,maxage=10h,dumponexit=true,loglevel=info"]
 
+
```
Comment (JustinPihony) [inline build.gradle]: We should document how to use this. Or at least point to the logback documentation

Comment (colinsheppard) [reply build.gradle]: Agreed with Justin, otherwise LGTM.

**Comment by JustinPihony** [comment]
test!

**Comment by JustinPihony** [comment]
I cleaned this up and added documentation - @REASY or @colinsheppard please double check the docs work with you :)

File: docs/developers.rst [hunk 2]
```diff
@@ -421,6 +423,8 @@ Now at the bottom, under NetworkSettings, locate IP Address of your docker conta
 
 
 
+.. _already configured: http://www.python.org/
```
Comment (irishwithaxe) [inline docs/developers.rst]: @JustinPihony, why it is python.org?
Should that link point to some configuration example?

Comment (JustinPihony) [reply docs/developers.rst]: Doh - copy and paste....thanks for the catch - will fix

**Comment by JustinPihony** [comment]
test!

**Pull request merged**